cli/fix/fix.go

```diff
@@ -95,7 +95,13 @@ func walk(moduleOrWorkspaceFile string) error {
 				return err
 			}
 			if d.IsDir() {
-				return nil
+				// Don't recurse into hidden directories such as .git or .ijwb
+				// (IntelliJ).
+				if strings.HasPrefix(d.Name(), ".") {
```

Comment:
seems like more generally we could probably use `.bazelignore` here? but this seems fine for now